chat.go

```diff
@@ -428,3 +428,78 @@ func (c *Client) UpdateUserChatColor(params *UpdateUserChatColorParams) (*Update
 
 	return update, nil
 }
+
+type SendChatMessageParams struct {
+	// The ID of the broadcaster whose chat room the message will be sent to
+	BroadcasterID string `query:"broadcaster_id"`
+
+	// The ID of the user sending the message. This ID must match the user ID in the user access token
+	SenderID string `query:"sender_id"`
+
+	// The message to send. The message is limited to a maximum of 500 characters.
+	// Chat messages can also include emoticons.
+	// To include emoticons, use the name of the emote.
+	// The names are case sensitive.
+	// Don’t include colons around the name (e.g., :bleedPurple:).
+	// If Twitch recognizes the name, Twitch converts the name to the emote before writing the chat message to the chat room
+	Message string `query:"message"`
+
+	// The ID of the chat message being replied to
+	ReplyParentMessageID string `query:"reply_parent_message_id"`
+}
+
+type ChatMessageResponse struct {
+	ResponseCommon
+
+	Data ManyChatMessages
+}
+
+type ManyChatMessages struct {
+	Messages []ChatMessage `json:"data"`
+}
+
+type ChatMessage struct {
+	// The message id for the message that was sent
+	MessageID string `json:"message_id"`
+
+	// If the message passed all checks and was sent
+	IsSent bool `json:"is_sent"`
+
+	// The reason the message was dropped, if any
+	DropReasons ManyDropReasons `json:"drop_reason"`
+}
+
+type ManyDropReasons struct {
+	Data DropReason
+}
+
+type DropReason struct {
+	// Code for why the message was dropped
+	Code string `json:"code"`
+
+	// Message for why the message was dropped
+	Message string `json:"message"`
+}
+
+// Requires an app access token or user access token that includes the user:write:chat scope.
+// If app access token used, then additionally requires user:bot scope from chatting user,
+// and either channel:bot scope from broadcaster or moderator status
+func (c *Client) SendChatMessage(params *SendChatMessageParams) (*ChatMessageResponse, error) {
+	if params.BroadcasterID == "" {
+		return nil, errors.New("error: broadcaster id must be specified")
+	}
+	if params.SenderID == "" {
+		return nil, errors.New("error: sender id must be specified")
+	}
+
+	resp, err := c.put("/chat/messages", &ManyChatMessages{}, params)
```

Comment:
Thats a post, not put.

Reply:
https://dev.twitch.tv/docs/api/reference/#send-chat-message

Reply:
Fixed